### PR TITLE
UI/header

### LINKED ISSUE #27: [FEAT]: ui/home - header
### Summary
NA
### Answer following questions?
1. Is your feature related to FrontEnd? Yes
2. Is your feature related to Backend? No
3. Is your feature related to API Integration? No
### Resources
image

### PR DESCRIPTION
**Pull request by sub1120**

### CODE REVIEW

File: src/app/layout.tsx [hunk 1]
```diff
@@ -22,6 +24,7 @@ export default function RootLayout({
     <html lang="en">
       <body className={inter.className}>
         <ThemeProvider attribute="class" defaultTheme="system" enableSystem>
+          <Header></Header>
```
Comment (shivamvijaywargi) [inline src/app/layout.tsx]: You can have a self closing tag.

File: src/app/layout.tsx [hunk 2]
```diff
@@ -1,8 +1,10 @@
 import type { Metadata } from "next";
 import { Inter } from "next/font/google";
 
+import "./globals.css";
```
Comment (shivamvijaywargi) [inline src/app/layout.tsx:4]: Double import?

**Review by shivamvijaywargi — CHANGES_REQUESTED** [review]
Please read the comments I have left.

File: src/app/layout.tsx (hunk 1 above)
Comment (sub1120) [reply src/app/layout.tsx]: Corrected, please review

File: src/app/layout.tsx (hunk 2 above)
Comment (sub1120) [reply src/app/layout.tsx:4]: Corrected, please review

**Review by shivamvijaywargi — APPROVED** [review]
Looks good

**Pull request merged**